gradle/libs.versions.toml

```
@@ -6,6 +6,8 @@ ktor = "2.3.11"
 mockk = "1.13.11"
 slf4j = "2.0.13"
 sonarqube = "4.4.1.3373"
+kotlin-coroutines-test = "1.8.1"
+koin-bom = "3.5.6"
```

Change this version name to koin, as it's the same as other Koin dependencies (it's not exclusive to BOM)

```suggestion
koin = "3.5.6"
```